index.js

```diff
@@ -131,6 +150,13 @@ module.exports = function (session) {
    */
 
   PGStore.prototype.query = function (query, params, fn) {
+    if (this.pgPool !== undefined) {
+      this.pgPool.query(query, params, function (err, res) {
+        if (fn) { fn(err, res && res.rows[0] ? res.rows[0] : false); }
+      });
+      return;
+    }
+
     if (!fn && typeof params === 'function') {
       fn = params;
       params = [];
```

Comment:
You're right about the parameter order, but pg-pool.query already handles this for us. See https://github.com/brianc/node-pg-pool/blob/master/index.js#L120